docs/recipes/simple_rag.adoc

```diff
@@ -0,0 +1,178 @@
+│         │ │        │ • 4.1. Working with clusters                       │ │
+│         │ │        │ • 4.2. Working with buckets                        │ │
+│         │ │        │ • 4.3. Working with scopes and collections         │ │
+│         │ │        │                                                    │ │
+│         │ │        │ • 4.3.1. Scopes                                    │ │
+│         │ │        │ • 4.3.2. Collections                               │ │
+│         │ │        │                                                    │ │
+│         │ │        │ • 4.4. Listing nodes                               │ │
+│         │ │        │ • 4.5. Reading and Writing `doc`uments             │ │
+│         │ │        │                                                    │ │
+│         │ │        │ • 4.5.1. Reading                                   │ │
+│         │ │        │ • 4.5.2. Mutating                                  │ │
+│         │ │        │ • 4.5.3. Remov                                     │ │
+│         │ │        │ ╭──────┬───────╮                                   │ │
+│         │ │ vector │ │    0 │ -0.00 │                                   │ │
+│         │ │        │ │  ... │  ...  │                                   │ │
+│         │ │        │ │ 1535 │ -0.02 │                                   │ │
+│         │ │        │ ╰──────┴───────╯                                   │ │
+│         │ ╰────────┴────────────────────────────────────────────────────╯ │
+╰─────────┴─────────────────────────────────────────────────────────────────╯
+```
+
+`vector enrich-text` will create a `vector doc` for each of the text chunks, these docs contain the original chunk of text along with the embedding generated from it.
+These docs are https://couchbase.sh/docs/#_a_note_on_data_format[formatted] so that they can be piped directly into https://couchbase.sh/docs/#_mutating[doc upsert] to store them in the active Couchbase Cluster.
+
+```
+👤 Charlie 🏠 remote in ☁️ RagChunks._default._default
+> ls | vector enrich-text | doc upsert
+Embedding batch 1/1
+╭───┬───────────┬─────────┬────────┬──────────┬─────────╮
+│ # │ processed │ success │ failed │ failures │ cluster │
+├───┼───────────┼─────────┼────────┼──────────┼─────────┤
+│ 0 │       152 │     152 │      0 │          │ remote  │
+╰───┴───────────┴─────────┴────────┴──────────┴─────────╯
+```
+
+We can see from the output of `doc upsert` that the 152 vector docs have been successfully uploaded to the active bucket named `RagChunks`.
+Next we use https://couchbase.sh/docs/#_vector_create_index[vector create-index] to create a vector index over our new vector docs:
+
+```
+👤 Charlie 🏠 remote in ☁️ RagChunks._default._default
+> vector create-index cbsh-docs-index vector 1536
+```
+
+See the previous link of a detailed explanation of this command.
+
+Now that we have craeted and indexed our vector docs we can take a look at the final command we will use: https://couchbase.sh/docs/#_ask[ask].
```

Comment:
craeted?  :)

Might be good to run a spell checker over these docs in an automated way.  Simon Murray did this for operator and it was actually quite a boon, as it forced us to consistently document technical terms.

Reply:
Corrected and raised an issue to do this: https://github.com/couchbaselabs/couchbase-shell/issues/442